services/launcher-dev/app/routes.py

```diff
@@ -13,70 +13,21 @@
 from app import db
 from app import utils
 from flask import request
+from threading import Lock
```

Comment:
The lock used here only assure that the docker service is thread-safe, but not synchronized between processes.